src/scripts/clipperUI/components/popover.tsx

```diff
@@ -25,19 +25,8 @@ class PopoverClass extends ComponentBase<{}, PopoverProps> {
 
 	handlePopoverLifecycle(element, isInitialized, context) {
 		if (!isInitialized) {
-			let popoverObj: any = {
-				content: this.props.content,
-				classNames: this.props.classNames,
-				arrowClassNames: this.props.arrowClassNames
-			};
-
-			let mainControllerElem = document.getElementById(Constants.Ids.mainController);
-			if (mainControllerElem) {
-				// We want to set the parent lower in the HTML hierarchy to avoid z-index issues relating to stacking contexts
-				popoverObj.parent = mainControllerElem;
-			}
-
-			this.refToPopper = new popperJS(document.getElementById(this.props.referenceElementId), popoverObj, {
+			let popperElement = this.generatePopperElement(Constants.Ids.mainController);
+			this.refToPopper = new popperJS(document.getElementById(this.props.referenceElementId), popperElement, {
 				placement: this.props.placement,
 				modifiersIgnored: this.props.modifiersIgnored,
```

Comment:
`modifiersIgnored` is deprecated. Full migration guide at https://github.com/FezVrasta/popper.js/issues/62